calcom/cal.com
# chore: Chore team metadata table + isOrganization migration from metadata

## What does this PR do?



## Type of change




## How should this be tested?

1. Seed the database 
2. Connect to DB 
3.  UPDATE "Team" SET "metadata" = '{"isOrganization": "true", "isOrganizationConfigured": "true", "orgAutoAcceptEmail": "example@example.com", "isOrganizationVerified": "false"}'
WHERE "id" in (1,2,3);
UPDATE "Team" SET "isOrganization" = true WHERE "id" in (1,2,3);
4. Run SQL above to set metadata 
5. Run migrations - ensure migrations have ran successfully and each team in the above query has a OrganizationSettings  table associated with it 


## Mandatory Tasks


## Checklist



- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
